Library/DiscUtils.Core/VirtualDisk.cs

```diff
@@ -523,7 +523,7 @@ public virtual byte[] GetMasterBootRecord()
 
             long oldPos = Content.Position;
             Content.Position = 0;
-            StreamUtilities.ReadFully(Content, sector, 0, Sizes.Sector);
```

Comment:
This call did not throw an exception on a short read before, but it will now. I believe it was as bug. A sector is a well defined chunk of data. How would a short read be correct here?